stable/gerrit/templates/deployment.yaml

```diff
@@ -0,0 +1,101 @@
+{{- $volumes := .Values.persistence.volumes }}
+{{- $releaseName := .Release.Name }}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "gerrit.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "gerrit.name" . }}
+    helm.sh/chart: {{ include "gerrit.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ include "gerrit.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "gerrit.name" . }}
+        release: {{ .Release.Name }}
+    spec:
```

Comment:
The ingress spec is not correct:
You should be able to remove the init container if you add this instead:

```yaml
securityContext:
  runAsUser: 1000
  fsGroup: 1000
  runAsNonRoot: true
```